packetbeat/beater/packetbeat.go

```diff
@@ -26,48 +26,48 @@ import (
 )
 
 // Beater object. Contains all objects needed to run the beat
-type Packetbeat struct {
-	Config      config.Config
-	CmdLineArgs CmdLineArgs
-	Pub         *publish.PacketbeatPublisher
-	Sniff       *sniffer.SnifferSetup
+type packetbeat struct {
+	config      config.Config
```

Comment:
Not sure about this change as Packetbeat can also be used as Library and it would be nice to at least have access tot he Config options. I sometimes have that problem with Golang libraries that most things are private and cannot be directly access which leads to coping code.